src/kirby/components/empty-state/empty-state.component.tns.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../scss/native-imports";
+
+.box {
+  width: 88%;
```

Comment:
Should be:
```suggestion
  width: 330;
```

Reply:
No, then it would be to wide for the smallest devices. I think instead it should be 88% of the small devices. Maybe it should have added a max-width.

Reply:
Remove width from {N} - we'll let the containing component decide for now

Reply:
I've removed the width. Not it looks like this:
![Screenshot_1562234005](https://user-images.githubusercontent.com/759680/60657917-6f1a1d00-9e52-11e9-84e8-f6f09cb86ad7.png)

Reply:
That's fine for now :)